udaykalra/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +90,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   /**
-   * Obtains parameter from Comments typing field.
+   * Gets count for comments to be shown.
```

Comment:
Would be helpful to point out that -1 is returned if an error occurs.

It would be better practice to then define a file-scoped constant that represents "no comment limit", instead of relying on users of this method to use the literal -1. This would be less error-prone and more resilient to refactoring, as well as easier to read.

Alternatively, and maybe this is the preferable approach, this method could return Integer instead of int, where nil is returned if there is no limit. Then you wouldn't need to define a special value.

Reply:
Changed to return null through Integer class.